src/main/java/moomoo/command/AddExpenditureCommand.java

```diff
@@ -33,6 +21,12 @@ public void execute(ScheduleList calendar, Budget budget, CategoryList categoryL
             throws MooMooException {
         super.execute(calendar, budget, categoryList, category, ui, storage);
 
+        ui.showAddExpenditureMessage();
+        String input = ui.readCommand();
+        String[] parts = input.split("-");
+        String expenditureName = parts[0];
+        Double amount = Double.parseDouble(parts[1]);
+
         for (int i = 0; i < categoryList.size(); i++) {
             if (categoryList.get(i).toString().equals(expenditureName)) {
```

Comment:
Think expenditureName should be categoryName. Also, can you allow user to input LocalDateTime and an actual expenditure name (for example laksa or chicken rice under food category)? Then change constructor for expenditure to add the name, amount and localdatetime.

Reply:
Okay noted will make this change